triggers/daml/Daml/Trigger.daml

```diff
@@ -43,7 +43,8 @@ module Daml.Trigger
  , fromCreated
  , fromArchived
  , CompletionStatus(..)
- , RegisteredTemplates(..)
+ , RegisteredTemplates(AllInDar, ..)
+ , AllInDar
```

Comment:
Since this is a reexport, the pattern `AllInDar` is already "bundled" so this line doesn't need to change

```suggestion
 , RegisteredTemplates(..)
```

https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/pattern_synonyms.html#import-and-export-of-pattern-synonyms